applied1/reservation.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Sage Bistro</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" href="_css/al1.css">
+    <link rel="stylesheet" href="themes/ctheme1.css">
+
+</head>
+
+<body>
+    <!-- Benjamin Cook
+     assignment?.html
+     INFO2439
+     Thoendel
+     9/4/2019
+-->
+    <div data-role="header">
+        <a data-ajax="false" href="../index.html" data-icon="home" rel="external">Home</a>
+        <p style="text-align:center;"><img src="photos/sagelogo.png" alt="Sage Bistro Logo"></p>
+    </div>
+    <div class="topnav">
+            <a data-ajax="false" href="menu.html">Menu</a>
+            <a data-ajax="false" href="reservation.html">Reservations</a>
+            <a data-ajax="false" href="survey.html">Survey</a>
+            <a data-ajax="false" href="map.html">Location</a>
+        </div>
+    <header>
+        <h1>Reservations</h1>
+    </header>
+    <p>HOURS OF OPERATION (DURING OPERATING DATES)
+        Lunch: Monday – Thursday: 10 a.m.–1 p.m.
+        Dinner: Monday – Thursday: 1 p.m.-12 a.m. (last seating at 11 p.m.)</p>
+    <div data-role="content">
+
+        <form method="post" action="https://www.mccinfo.net/webresources/ogi-bin/formmail.asp">
```

Comment:
I wasn't expecting you to write a form for this functionality. Additionally - that formmail.asp doesn't work - which is why I pointed you to a php file I host on my server in Assignment 4. You have a great form here, you just don't have the backend to support it.